mmv1/third_party/terraform/utils/config.go.erb

```diff
@@ -78,6 +83,8 @@ type providerMeta struct {
 	ModuleName string `cty:"module_name"`
 }
 
+var providerVersion string
```

Comment:
Instead of a global var, let's set this during `providerConfigure` in `provider.go`: https://github.com/borg-land/magic-modules/blob/cloudrun-versioning-annotations/mmv1/third_party/terraform/utils/provider.go.erb#L505

We can share code w/ the useragent creation in that file.

That should also sidestep the tgc failure (config.go is used there, provider.go isn't)

Reply:
Fixed